host/README.md

```diff
@@ -146,8 +153,7 @@ If everything is successful, you should see a screen as shown below.
 
 ![](images/vai_c_output_caffe.png)
 
-A new model file (e.g. `dpu_resnet50_0.elf`) should appear in your working 
-directory; this is the model file that can be deployed on the board.
+A new model file (e.g. `dpu_resnet50_0.elf`) should appear in your `generated/build/` directory; this is the model file that can be deployed on the board.
```

Comment:
Better to keep 2 lines instead of 1 line to better track the changes.
Is there a reason why we want to create the `generated/build` directory? 
If this is not necessary, I prefer it to have the original file path - so users won't be surprised.

Reply:
Added back line break

The reason to have the generated and build folder is the same as for all compilation flows. If you run the compile.sh without it, you will have a dozen files shown in your git diff and you will have trouble fishing out your real changes. You would have to `reset --hard` to get rid of them easily. With this change, all you have to do is delete the generated folder to clean up. On top of that, generated/ is added to the gitignore, so the 'trash' will not show up on your diff.

In addition, the only file you're looking for is the mymodel.elf after a compilation. It's much easier to copy it from the build folder than to find it in your host folder with a dozen more files with similar names.

It is not a necessary change, but it is a nice quality of life improvement :)
I understand if you will not want to merge it

Reply:
Thanks for the improvement. I think this is okay. My last suggestion is, we don't make too many nested folders, so you can have a single folder called `build` (instead of `generated/build`), and put every generated files into it, that will be all good. Any reason not to do that way?

Reply:
Just that the .elf file will be harder to find among all the other files, but I can live with that.